mautic/mautic
# Move Web Push & SMS Config from global config to the plugins page

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR moves the Twilio & One Signal configuration from the global configuration page into appropriate plugin/integration forms. It also adds a new integration form modification event.

[//]: # ( As applicable: )

#### Steps to test this PR:
1. Setup SMS and Web Config actions in a campaign, and verify that they are working.
2. Apply the PR
3. Check that all the configuration settings were moved into the integration forms.
4. Retrigger the campaign actions and ensure they still function properly.
